_includes/includes/ui/keyboard-details.php

```diff
@@ -142,11 +145,10 @@ protected static function WriteWebBoxes() {
         if (!isset($lang)) $lang = 'en';
         $url = "{$KeymanHosts->keymanweb_com}/#$lang,Keyboard_" . self::$keyboard->id;
         $description = htmlentities(self::$keyboard->name);
-          return <<<END
+        return <<<END
           <div class="download download-web">
-          <a class="download-link" $embed_target href='$url'>Use keyboard online</a>
-          <div class="download-description">Use $description in your web browser. No need to install anything.</div>
-        </div>
+            <a class="download-link" $embed_target href='$url'>Full online editor</a>
+          </div>
```

Comment:
Can you note  the rationale why the blue button changed 
from "*Use keyboard online*" 
to "*Full online editor*"? (and removed the subtext)

I don't have a strong preference. (though does "Full online editor" imply a lot more features than what KeymanWeb offers?)

Reply:
Rationale -- I agree that 'full online editor' implies a future version of keymanweb.com which has awesomesauce which we just don't currently have. I wanted to differentiate between the one line test box and the more fully featured keymanweb.com, and am open to other ideas.

I thought "Use keyboard online" was confusing because with this new feature the user will already be using the keyboard online. I removed the subtext because it seemed self-evident as it was below the existing keyboard. (I could add the subtext back again for pages which don't have the preview)